components/azure/azureml/aml-deploy-model/src/deploymentconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "containerResourceRequirements": {
+       "cpu": 1,
+       "memoryInGB": 1
+    },
+    "computeType": "ACI",

```

Comment:
Is it possible to include deployment JSON for AKS as well and control the deployment to either ACI or AKS from the input?

Reply:
Sushant do you mean deploy them locally to the AKS? I have seen config like 
```{
    "computeType": "LOCAL"
}
```
But I am not sure this is what you like?

Reply:
https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#azure-kubernetes-service-deployment-configuration-schema

Reply:
Aha, I see. Do you mean we want to actually use AKS as well. I think we can add code like this as commented out code chunk or we can add a new deploymentconfig.json
```
{
    "computeType": "aks",
    "autoScaler":
    {
        "autoscaleEnabled": true,
        "minReplicas": 1,
        "maxReplicas": 3,
        "refreshPeriodInSeconds": 1,
        "targetUtilization": 70
    },
    "dataCollection":
    {
        "storageEnabled": true
    },
    "authEnabled": true,
    "containerResourceRequirements":
    {
        "cpu": 0.5,
        "memoryInGB": 1.0
    }
}
```

Reply:
Changed.